infoschema/tables.go

```diff
@@ -803,7 +803,7 @@ const (
 	uniqueKeyType  = "UNIQUE"
 )
 
-// See https://dev.mysql.com/doc/refman/5.7/en/table-constraints-table.html
+// dataForTableConstraints See https://dev.mysql.com/doc/refman/5.7/en/table-constraints-table.html
```

Comment:
dataForTableConstraints constructs data for table information_schema.constraints.